Chandra/cmd_states/cmd_states.py

```diff
@@ -107,6 +108,23 @@ def add_trans(date=date, **kwargs):
     return add_trans
 
 
+def _make_pitch_cmds(cmds, sample_time=5000):
+    """
+    Add cmds to clock out pitch for any long states
+    """
+    cmd_times = np.array(DateTime([cmd['date'] for cmd in cmds]).secs)
+    durations = cmd_times[1:] - cmd_times[0:-1]
+    long_dur = durations > sample_time
+    pitch_cmd_times = []
+    for cmd_time, dur in zip(cmd_times[long_dur], durations[long_dur]):
```

Comment:
Thanks.  I also wondered, should we add these pitch-clocking mock-commands at the end of the commands that we have available?  Or do we consider the last command time the end of evaluation of the model?